benchmarks/triton_kernels_benchmark/benchmark_testing.py

```diff
@@ -163,9 +163,11 @@ def extract_kernels(funcs):
     # for correct registration of kernels.
     # For details: https://github.com/pytorch/pytorch/issues/144778
     kernels = [kernel for kernel in kernels if kernel != []]
-    assert len(kernels) == n_repeat, (
-        f"the profiling number not match; {n_repeat=}, {kernels=}, \n" +
-        f"top functions by xpu_time:\n {prof.key_averages(group_by_stack_n=5).table(sort_by='xpu_time')}")
+    relax_profiling_data_check = os.getenv("TRITON_RELAX_PROFILING_CHECK", "0") == "1"
+    if not (len(kernels) >= n_repeat - 1 if relax_profiling_data_check else len(kernels) == n_repeat):
+        raise AssertionError(
+            f"the profiling number not match; {n_repeat=}, {kernels=}"
+            f"top functions by xpu_time:\n {prof.key_averages(group_by_stack_n=5).table(sort_by='xpu_time')}")
```

Comment:
`\n` was removed, you need to visually separate the concatenated strings, for example:
```suggestion
            f"the profiling number not match; {n_repeat=}, {kernels=}, "
            f"top functions by xpu_time:\n {prof.key_averages(group_by_stack_n=5).table(sort_by='xpu_time')}")
```

Reply:
Fixed